examples/apiserver/server/main.go

```diff
@@ -29,7 +29,6 @@ func main() {
 	serverRunOptions := apiserver.NewServerRunOptions()
 
 	// Parse command line flags.
-	serverRunOptions.AddUniversalFlags(pflag.CommandLine)
```

Comment:
instead of removing, maybe this should be changed to `serverRunOptions.GenericServerRunOptions.AddUniversalFlags(pflag.CommandLine)`?